### DISCUSSION TITLE
Bolls.life notes Sync

### DISCUSSION THREAD

**Discussion by micahmorgan1** [tim-hub/obsidian-bible-reference#121] [Ideas]
This may be slightly off topic, but through this project I was introduced to the Bolls.life project which seems fantastic.  I was just curious what the feasibility of a plugin syncing notes from bolls.life to obsidian would be.  (similar to the readwise sync).  I think this combined with the bible reference app would be a great pair, where you could read the bible and take notes in the bolls.life app (something designed for reading the Bible), but have your notes synced back to obsidian for cross-referencing/tagging from obsidian bible reference, and lesson prep. I'm not a coder so this is beyond me, but I was just curious what the feasibility of this would be and if there are other similar options out there for syncing the Bible reading experience with obsidian notes.

**Comment by tim-hub** [comment]
It is good to have I believe, but I have to say this is hard to do, because I am not very famaliar with bolls life front-end, and I don't believe it is open source as well. 

Have to put this as low priority at current stage.
